_starter-kit/_patterns/03-components/article/article.twig

```diff
@@ -1,25 +1,18 @@
 {% if not classes %}
   {% set classes = [] %}
 {% endif %}
-{% set classes =
-  classes|merge(['article'])|join(' ')|trim
-%}
-
-<article{{- attributes.addClass(classes)|raw -}}>
+{% set classes = classes | merge(['article']) | join(' ') | trim %}
+<article{{-attributes.addClass(classes)|raw-}}>
   {% block title %}
-    {% include '@components/page-title/page-title.twig' with {
-      'page_title': article_page_title
-    } %}
+    {{ title|raw }}
```

Comment:
Why'd you remove the call to the page-title component?  Now this will just print out plain text with no wrapper element.

Reply:
This might be a copy paste error. Will revert.